Services/Certificate/classes/Migration/class.ilCertificateMigrationValidator.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/* Copyright (c) 1998-2018 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+/**
+ * @author  Niels Theen <ntheen@databay.de>
+ */
+class ilCertificateMigrationValidator
+{
+	/**
+	 * @var ilSetting
+	 */
+	private $certificateSettings;
+
+	/**
+	 * @param ilSetting $certificateSettings
+	 */
+	public function __construct(ilSetting $certificateSettings)
+	{
+		$this->certificateSettings = $certificateSettings;
+	}
+
+	/**
+	 * @param ilObjUser $user
+	 * @return bool
+	 */
+	public function isMigrationAvailable(ilObjUser $user)
+	{
+		$userCreationDate = $user->getCreateDate();
```

Comment:
@legionth / @qdittrich : Maybe other *preconditions* should be added here as well